mlfinlab/feature_importance/cfi.py

```diff
@@ -0,0 +1,126 @@
+"""
+An Implementation of the Clustered Feature Importance algorithm described by
+Dr Marcos Lopez de Prado in 'Clustered Feature Importance (Presentation Slides)'
+(https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3517595)
+"""
+
+#Imports
+import numpy as np
+import pandas as pd
+import mlfinlab as ml
+from sklearn.metrics import log_loss
+from scipy.cluster.hierarchy import linkage, fcluster
+from mlfinlab.codependence import variation_of_information_score
+
+def get_VI_matrix(df : pd.DataFrame):
+    '''
+    Get Variation of Information (VI) matrix.
+
+    :param df: (pd.DataFrame) of features
+    :return: (pd.DataFrame) of VI matrix
+    '''
+    VI = pd.DataFrame(index=df.columns)
+    features_cols = df.columns.values
+    for col0 in features_cols:
+        vi = []
+        for col1 in features_cols:
+            x = df[col0].values
+            y = df[col1].values
+            vi.append(variation_of_information_score(x,y,n_bins=None,normalize=True))
+        VI[col0] = vi
+    return VI
+
+def cluster_features(X : pd.DataFrame, C : int = None):
```

Comment:
Can we change C parameter name to n_clusters as it defined in clustered_feature_importance?

Reply:
I agree that makes total sense. I kept it so because it was just used as a helper function.